netcore/corefx-restore.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp30</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+  </PropertyGroup>
+  <PropertyGroup>
+    <SystemCompositionVersions>$(MicrosoftPrivateCoreFxNETCoreAppVersion.Replace("4.6.0", "1.3.0"))</SystemCompositionVersions>
+    <SystemNumericsTensorsVersion>$(MicrosoftPrivateCoreFxNETCoreAppVersion.Replace("4.6.0", "0.2.0"))</SystemNumericsTensorsVersion>
+    <SystemDataSqlClientVersion>$(MicrosoftPrivateCoreFxNETCoreAppVersion.Replace("4.6.0", "4.7.0"))</SystemDataSqlClientVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Private.CoreFx.NETCoreApp" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.Json" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.CodeDom" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.ComponentModel.Composition" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.ComponentModel.Composition.Registration" Version="$(MicrosoftPrivateCoreFxNETCoreAppVersion)" />
+    <PackageReference Include="System.Composition.AttributedModel" Version="$(SystemCompositionVersions)" />
```

Comment:
I wonder why these packages have different versions in CoreFX?

Reply:
For many historical and stupid reasons. It's possible for CoreFX folks to publish one NuGet manifest with the right version number for consumption from Mono. This would shrink to 1-2 lines then. Alternatively they can publish it as downloadable archive.